src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -328,8 +330,8 @@ Microsoft.CodeAnalysis.Semantics.IIsPatternExpression
 Microsoft.CodeAnalysis.Semantics.IIsPatternExpression.Expression.get -> Microsoft.CodeAnalysis.IOperation
 Microsoft.CodeAnalysis.Semantics.IIsPatternExpression.Pattern.get -> Microsoft.CodeAnalysis.Semantics.IPattern
 Microsoft.CodeAnalysis.Semantics.IIsTypeExpression
-Microsoft.CodeAnalysis.Semantics.IIsTypeExpression.IsType.get -> Microsoft.CodeAnalysis.ITypeSymbol
 Microsoft.CodeAnalysis.Semantics.IIsTypeExpression.IsNotTypeExpression.get -> bool
```

Comment:
Anyone know why those unrelated changes are happening? I assume this file is maintained in a sorted order, but it looks like the previous checked-in version was not.

Reply:
Likely that some previous change was hand crafted and not via code fix, and it broke the ordering.

Reply:
I have also seen merge resolution introduce such ordering issues.